src/execute/oas3/style-serializer.js

```diff
@@ -6,6 +6,11 @@ export function encodeDisallowedCharacters(str, { escape } = {}, parse) {
   if (typeof str === 'number') {
     str = str.toString();
   }
+
+  if (typeof str === 'object' && !Array.isArray(str)) {
```

Comment:
If you really want to just support POJOs here (Plain Old JavaScript Objects), and not `null`, you have to do the followig.

```suggestion
  if (str !== null && typeof str === 'object' && !Array.isArray(str)) {
```